web/admin/application/library/IvozProvider/Klear/Filter/CompanyServices.php

```diff
@@ -47,9 +48,22 @@ public function setRouteDispatcher(KlearMatrix_Model_RouteDispatcher $routeDispa
             ["BrandService.brand = " . $currentBrandyId]
         );
 
+        $availableServices = [
+            Service::DIRECT_PICKUP,
+            Service::GROUP_PICKUP,
+            Service::VOICEMAIL,
+            Service::RECORD_LOCUTION,
+            Service::CLOSE_LOCK,
+            Service::OPEN_LOCK,
+            Service::TOGGLE_LOCK,
+        ];
+
         $brandServicesIds = array();
         foreach ($brandServices as $brandService) {
-            array_push($brandServicesIds, $brandService->getServiceId());
+            $serviceIden = $brandService->getService()->getIden();
+            if (in_array($serviceIden, $availableServices)) {
```

Comment:
in_array causes unexpected results without $strict flag on, we should use it by default => in_array ( mixed $needle , array $haystack , true)